<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BF4BH2</class>
 <widget class="QMainWindow" name="BF4BH2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BF4BotHunter2</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/res/bf4bh.ico</normaloff>:/res/res/bf4bh.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="uileSearch">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>https://battlelog.battlefield.com/bf4/...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="uibSearch">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="uitvMain">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="uipb">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="uib0">
        <property name="text">
         <string>Кнопка 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="uileFilter">
        <property name="placeholderText">
         <string>Поиск в таблице</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="uib1">
        <property name="text">
         <string>Кнопка 2</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="uim0">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="action_URL"/>
    <addaction name="uiaUpdate"/>
    <addaction name="uiaRow"/>
    <addaction name="separator"/>
    <addaction name="uiaAbout"/>
    <addaction name="uiaQuit"/>
   </widget>
   <widget class="QMenu" name="uimPreset">
    <property name="title">
     <string>Предустановки</string>
    </property>
   </widget>
   <addaction name="uim0"/>
   <addaction name="uimPreset"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="uiaQuit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="uiaAbout">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="uiaRow">
   <property name="text">
    <string>Подогнать строки по содержанию</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="uiaUpdate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Автообновление данных</string>
   </property>
  </action>
  <action name="action_URL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Поиск по URL</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
